src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -112,26 +134,19 @@ describe("Editor Page Component", () => {
         });
 
         const wrapper = createComponent(store, props);
-        const editorPage = wrapper.find(EditorPage).childAt(0) as ReactWrapper<IEditorPageProps, IEditorPageState>;
+        const editorPage = wrapper.find(EditorPage).childAt(0);
 
         const partialProject = {
             id: testProject.id,
             name: testProject.name,
         };
 
-        const expectedAssetMetadtata: IAssetMetadata = {
-            asset: {
-                ...testAssets[0],
-                state: AssetState.Visited,
-            },
-            regions: [MockFactory.createMockRegion()],
-            timestamp: null,
-        };
+        const expectedAssetMetadtata: IAssetMetadata = getMockAssetMetadata(testAssets);
 
         setImmediate(() => {
             expect(editorPage.props().project).toEqual(expect.objectContaining(partialProject));
             expect(editorPage.state().assets.length).toEqual(testAssets.length);
-            expect(editorPage.state().selectedAsset).toEqual(expectedAssetMetadtata);
+            expect(editorPage.state().selectedAsset).toMatchObject(expectedAssetMetadtata);
```

Comment:
what's the difference?

Reply:
See above explanation with string matching